### PR TITLE
reliablility

### PR DESCRIPTION
**Pull request by stoeffel**
* make global specs in module description work
* allow no newline between specs
* cleanup
* add `Test.todo` when we a module doesn't contain verified examples

### CODE REVIEW

File: src/VerifyExamples/Ast.elm [hunk 1]
```diff
@@ -1,107 +1,117 @@
...
+    | Function String String
     | Type String
 
 
-isTest : Ast -> Bool
-isTest x =
-    case x of
-        Assertion _ ->
-            True
+type alias Groupped =
```
Comment (eeue56) [inline src/VerifyExamples/Ast.elm]: Only one p here but nbd

File: src/VerifyExamples/Ast.elm [hunk 2]
```diff
@@ -1,107 +1,117 @@
...
+    { functions : List Ast
+    , imports : List Ast
+    , types : List Ast
```
Comment (eeue56) [inline src/VerifyExamples/Ast.elm]: It seems to me that having these stored as AST will lead to boilerplate and cases that are impossible to reach. How about turning the AST for each into a dedicated token at this point?

File: src/VerifyExamples/Ast.elm [hunk 3]
```diff
@@ -1,107 +1,117 @@
...
-    IAst.convert
-        { maybeExpression = Assertion
-        , arrowPrefixed = Expectation
-        , importPrefixed = Import
-        , typePrefixed = Type
-        , function = LocalFunction
-        }
+setFunctions : Groupped -> List Ast -> Groupped
+setFunctions groupped functions =
+    { groupped | functions = functions }
```
Comment (eeue56) [inline src/VerifyExamples/Ast.elm]: Shouldn't these be flipped? i.e groupped last?

File: src/VerifyExamples/Ast.elm (hunk 2 above)
Comment (stoeffel) [reply src/VerifyExamples/Ast.elm]: totally. that's way better

File: src/VerifyExamples/Ast.elm (hunk 3 above)
Comment (stoeffel) [reply src/VerifyExamples/Ast.elm]: yes, I know. the pipeline looks so nice though :-)

**Comment by stoeffel** [comment]
everything addressed

**Pull request merged**